boto/configservice/__init__.py

```diff
@@ -30,7 +30,7 @@ def regions():
     :rtype: list
     :return: A list of :class:`boto.regioninfo.RegionInfo`
     """
-    from boto.kms.layer1 import ConfigServiceConnection
```

Comment:
If you tried ``connect_to_region()``, I believe so. But since there was only one region ``us-east-1`` you could work around it with ``connect_configservice()``